src/sentry/lang/javascript/processor.py

```diff
@@ -46,6 +45,9 @@
 BASE64_SOURCEMAP_PREAMBLE = "data:application/json;base64,"
 BASE64_PREAMBLE_LENGTH = len(BASE64_SOURCEMAP_PREAMBLE)
 UNKNOWN_MODULE = "<unknown module>"
+# Names that do not provide any reasonable value, and that can possibly obstruct
+# better available names. In case we encounter one, we fallback to current frame fn name if available.
+useless_fn_names = ["<anonymous>", "__webpack_require__", "__webpack_modules__"]
```

Comment:
```suggestion
USELESS_FN_NAMES = ["<anonymous>", "__webpack_require__", "__webpack_modules__"]
```

I think these are allcaps by convention